src/index.ts

```diff
@@ -169,13 +169,31 @@ const extension: JupyterFrontEndPlugin<void> = {
     const _trace = console.trace;
     const _table = console.table;
 
+    // https://stackoverflow.com/a/11616993
```

Comment:
Wondering whether we want to take this approach though.

As mentioned  in the SA answer:

> The replacer in this example is not 100% correct (depending on your definition of "duplicate"). In the following case, a value is discarded:

This indeed discards some of the elements:

![image](https://user-images.githubusercontent.com/591645/132859748-f7486c99-7dd8-4cf1-980d-7f187d40aab5.png)

Reply:
Added a warning message informing the user about the possibility of missing attributes